include/units.h

```diff
@@ -4854,9 +4854,53 @@ namespace std
 		{
 			return units::unit_t<Units, T, NonLinearScale>(std::numeric_limits<T>::lowest());
 		}
+		static constexpr units::unit_t<Units, T, NonLinearScale> epsilon()
+		{
+			return units::unit_t<Units, T, NonLinearScale>(std::numeric_limits<T>::epsilon());
+		}

```

Comment:
Since this is epsilon near 1.0, I feel like it's not unit-invariant the same way `lowest()` is.

Reply:
I am not sure, it depends on the underlying type, or I don't understand something,

Reply:
My thought here is I can’t think of a case where you’d want epsilon that wasn’t better served by multiplying by `std::numeric_limits<double>::epsilon()`. Epsilon is the smallest increment you add to 1.0 to get a different number, but with units, 1.0 what? I can find the epsilon WRT `1.0_mm` by multiplying by double-epsilon, but I can’t think of a use case in which the number isn’t magic. Does that make sense?

Reply:
Like if I change `D` between meters and millimeters then `1.0_mm + std::numeric_limits<D>::epsilon()` would change truthiness, I think.